Assets/Scripts/Enemy/EnemyService.cs

```diff
@@ -7,15 +8,17 @@ public class EnemyService : MonoSingletongeneric<EnemyService>
     {
         public List<GameObject> EnemyTankViews;
         public List<Vector3> EnemyTankPositions;
-        public List<GameObject> InstantiatedEnemyTanks;
+        EnemyPoolService enemyPoolService;
```

Comment:
service should be a singleton

Reply:
Solved by using Instance of pool service